Cargo.toml

```diff
@@ -10,7 +10,9 @@ keywords = ["gstd", "smart-contracts", "gear-tech", "gear-foundation"]
 categories = ["api-bindings", "no-std"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", rev = "946ac47" }
+gstd = { git = "https://github.com/gear-tech/gear.git", rev = "e9b6d89" }
```

Comment:
gstd's default is: `default = ["panic-handler"]`
so your feature below has no affect
you must import gstd with `default-features = false`